build.gradle

```diff
@@ -53,25 +71,27 @@ allprojects {
     }
 
     ext {
-        testcontainersVersion = "1.15.3"
-        slf4jVersion = "1.7.30"
+        testcontainersVersion = "1.16.0"
+        zstdVersion = "1.5.0-4"
+        junitVersion = "5.7.2"
     }
 
     dependencies {
-        implementation name: 'kafka-storage-api-3.0.0-SNAPSHOT'
-        implementation name: 'kafka-clients-3.0.0-SNAPSHOT'
-        implementation 'org.slf4j:slf4j-api:1.7.30'
-        testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.7.2'
-        testImplementation 'org.junit.jupiter:junit-jupiter-api:5.7.2'
+        implementation name: 'kafka-storage-api-3.1.0-SNAPSHOT'
+        implementation name: 'kafka-clients-3.1.0-SNAPSHOT'
+        implementation 'org.slf4j:slf4j-api:1.7.32'
+        implementation "com.github.luben:zstd-jni:$zstdVersion"
```

Comment:
this dependency must be transient. kafka uses the same library

Reply:
Done.